src/display/annotation_layer.js

```diff
@@ -530,6 +541,92 @@ var TextWidgetAnnotationElement = (
 
 /**
  * @class
+ * @alias CheckboxWidgetAnnotationElement
+ */
+var CheckboxWidgetAnnotationElement =
+    (function CheckboxWidgetAnnotationElementClosure() {
+  function CheckboxWidgetAnnotationElement(parameters) {
+    WidgetAnnotationElement.call(this, parameters);
+  }
+
+  Util.inherit(CheckboxWidgetAnnotationElement, WidgetAnnotationElement, {
+    /**
+     * Render the checkbox widget annotation's HTML element
+     * in the empty container.
+     *
+     * @public
+     * @memberof CheckboxWidgetAnnotationElement
+     * @returns {HTMLSectionElement}
+     */
+    render: function CheckboxWidgetAnnotationElement_render() {
+      this.container.className = 'checkboxWidgetAnnotation';
+
+      if (this.renderInteractiveForms) {
+        var element = document.createElement('input');
+        element.type = 'checkbox';
+        element.id = this.data.fullName;
+        if (this.data.fieldValue) {
+          element.checked = true;
+        }
+        this.container.appendChild(element);
+        element = document.createElement('label');
```

Comment:
Let's add an empty line before this line to make this a bit more readable as a new block of code.